lib/Http/Middleware/ProvisioningMiddleware.php

```diff
@@ -70,9 +70,15 @@ public function beforeController($controller, $methodName) {
 		}
 		try {
 			$this->provisioningManager->provisionSingleUser($configs, $user);
+			$password = $this->credentialStore->getLoginCredentials()->getPassword();
```

Comment:
huh, so `\OCP\Authentication\LoginCredentials\ICredentials::getPassword` lies? Time to fix that in server to specify a nullable string

Reply:
@miaulalala could you please take care of this?

Reply:
Done here https://github.com/nextcloud/server/pull/31377/files